packages/expo-linking/ios/ExpoLinkingModule.swift

```diff
@@ -0,0 +1,31 @@
+import ExpoModulesCore
+
+let onURLReceivedNotification = Notification.Name("onURLReceived")
```

Comment:
`"onURLReceived"` is already used in five places in this file – let's make it a global const